src/components/containers/victory-cursor-container.js

```diff
@@ -0,0 +1,123 @@
+/*eslint no-magic-numbers: ["error", { "ignore": [0, 1, 5, 10] }]*/
+import PropTypes from "prop-types";
+import React from "react";
+import { VictoryContainer, VictoryLabel, Line, Selection, Helpers } from "victory-core";
+import { isNumber } from "lodash";
+import CursorHelpers from "./cursor-helpers";
+
+export const cursorContainerMixin = (base) => class VictoryCursorContainer extends base {
+  static displayName = "VictoryCursorContainer";
+  static propTypes = {
+    ...VictoryContainer.propTypes,
+    defaultCursorValue: PropTypes.oneOfType([PropTypes.number, PropTypes.object]),
+    dimension: PropTypes.oneOf(["x", "y"]),
+    labelComponent: PropTypes.element,
+    labels: PropTypes.func,
+    onChange: PropTypes.func,
+    standalone: PropTypes.bool
+  };
+  static defaultProps = {
+    ...VictoryContainer.defaultProps,
+    labelComponent: <VictoryLabel/>,
+    cursorComponent: <Line/>
+  };
+
+  static defaultEvents = [{
+    target: "parent",
+    eventHandlers: {
+      onMouseLeave: () => {
+        return [];
+      },
+      onMouseMove: (evt, targetProps) => {
+        const mutations = CursorHelpers.onMouseMove(evt, targetProps);
+
+        if (mutations.id !== this.mouseMoveMutationId) { // eslint-disable-line
+          this.mouseMoveMutationId = mutations.id; // eslint-disable-line
+          return mutations.mutations;
+        }
+
+        return [];
+      }
+    }
+  }];
+
+  getCursorPosition(props) {
+    const { cursorPosition, defaultCursorValue, dimension } = props;
+    if (cursorPosition) { return cursorPosition; }
+
+    if (isNumber(defaultCursorValue)) {
+      return {
+        x: 0,
+        y: 0,
+        [dimension]: defaultCursorValue
+      };
+    }
+
+    return defaultCursorValue;
+  }
+
+  getCursorElements(props) {
+    const {
+      scale, domain, dimension, labelComponent, labels, cursorComponent
+    } = props;
+    const cursorPosition = this.getCursorPosition(props);
+
+    if (!cursorPosition) { return []; }
+
+    const newElements = [];
+    const domainCoordinates = Selection.getDomainCoordinates(scale, domain);
+
+    const cursorCoordinates = {
```

Comment:
totally understand. i think i'll keep it this way though, because it handles getting the position when `defaultCursorValue` (which can be both number and object) is used (i.e. there is no `onMouseMove`).